rpcd/playbooks/roles/rpc_maas/defaults/main.yml

```diff
@@ -464,6 +464,7 @@ maas_pip_packages:
   - rackspace-monitoring-cli
   - ipaddr
   - swift
+  - waxeye
```

Comment:
This failed to install on my AIO due to my locked down pip config pointing to the repo_server. We would either have to make sure the wheel is built in the repo_server, or use the ``--isolated`` flag to install the package.